packages/aws-cdk-lib/aws-autoscaling/lib/auto-scaling-group.ts

```diff
@@ -2252,11 +2284,92 @@ export class HealthCheck {
   private constructor(public readonly type: string, public readonly gracePeriod?: Duration) { }
 }
 
+/**
+ * Heath checks base options
+ */
+interface HealthChecksBaseOptions {
+  /**
+   * Specified the time Auto Scaling waits before checking the health status of an EC2 instance that has come into service
+   * and marking it unhealthy due to a failed health check.
+   *
+   * @default Duration.seconds(0)
+   * @see https://docs.aws.amazon.com/autoscaling/ec2/userguide/health-check-grace-period.html
+   */
+  readonly grace?: Duration;
```

Comment:
The `grace` has to be specified in the existing ELB options, but this is not necessary now. (I actually deployed it successfully).

https://github.com/aws/aws-cdk/blob/main/packages/aws-cdk-lib/aws-autoscaling/lib/auto-scaling-group.ts#L2221-L2226